webpack.config.js

```diff
@@ -119,7 +119,10 @@ const webpackConfig = ( mode ) => {
 						sourceMap: false,
 						cache: true,
 						terserOptions: {
-							keep_fnames: /__|_x|_n|_nx|sprintf/,
+							// We preserve function names that start with capital letters as
+							// they're _likely_ component names, and these are useful to have
+							// in tracebacks and error messages.
```

Comment:
This is slick!

Reply:
Indeed, that's gonna be super useful to debug issue reported through support too!